Dockerfile

```diff
@@ -15,5 +15,10 @@ RUN curl -Lfs "https://github.com/openshift/rosa/releases/download/$(curl -Lfs h
 RUN curl -LSs -o /usr/local/bin/opm "https://github.com/operator-framework/operator-registry/releases/download/$(curl -Lfs https://api.github.com/repos/operator-framework/operator-registry/releases/latest \
     | jq -r .tag_name)/linux-amd64-opm" \
     && chmod 0755 /usr/local/bin/opm
+RUN curl -Ls https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz | \
+    tar -zx && \
+    mv oc /usr/local/bin && \
+    mv kubectl /usr/local/bin && \
```

Comment:
`kubectl` is already installed. We do not use `oc` in pipelines, or do you use it?

Reply:
good point, thanks! I missed the fact the kubectl is already installed. I will see if I can use `kubectl` rather than `oc` - it is used in this PR for logging into cluster, but I can either use `kubectl login` instead or check whether API URL uses valid certificates without logging into cluster - should be doable.

Reply:
hupst, `kubectl login` does not exists so I will simply check API URL  via `curl` - the same way I do for console URL

Reply:
So installing neither oc nor kubectl is needed.